pages/roadmap.md

```diff
@@ -45,7 +39,15 @@ The goal of this roadmap is to give you a big picture. Here’s a list of our ke
 
 ## Kotlin roadmap by subsystem
 
-This table describes the biggest projects that we’re working on.
+You can view the biggest projects that we’re working on the [YouTrack board](https://youtrack.jetbrains.com/agiles/153-1251/current) and in the [Roadmap details](#roadmap-details) table.
```

Comment:
It seems like the other one "on" preposition is omitted ("working on on"). 
But it's better to rewrite the sentence to escape preposition doubling, provide a better reading, and relate it to the heading.

>The significant projects that we're working on are sorted into several subsystems. You can view them on the YouTrack board and in the Roadmap details table.

>We have sorted the significant projects that we're working on into several subsystems. You can view them on the YouTrack board and in the Roadmap details table.

Reply:
We've updated with copyeidtors: 
> To view the biggest projects we're working on, visit the [YouTrack board](https://youtrack.jetbrains.com/agiles/153-1251/current) or the [Roadmap details](#roadmap-details) table.